### PR TITLE
Moved some defines where they belong

### PR DESCRIPTION
**Pull request by DamianX**

### CODE REVIEW

**Comment by MadmanMartian** [comment]
but the mining scanner uses those defines

**Comment by DamianX** [comment]
nope, it doesn't

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -669,7 +669,7 @@ obj/item/asteroid/basilisk_hide/New()
 			if(Adjacent(M))
 				//Climb in
 				visible_message("<span class = 'warning'>\The [src] burrows itself into \the [M]!</span>")
-				M.rockernaut = istype(src, /mob/living/simple_animal/hostile/asteroid/rockernaut/boss)?2:TRUE
+				M.rockernaut = istype(src, /mob/living/simple_animal/hostile/asteroid/rockernaut/boss) ? ITS_A_BIG_ROCKERNAUT_WOW : YES_THERE_IS_A_ROCKERNAUT
```
Comment (ShiftyRail) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: The defines are undef'd in another file

**Comment by ShiftyRail** [comment]
RIP `ITS_A_BIG_ROCKERNAUT_WOW`, best define of 2018

**Pull request merged**